packages/wonder-blocks-form/src/components/text-field.tsx

```diff
@@ -95,6 +106,10 @@ type Props = AriaProps & {
      * Custom styles for the input.
      */
     style?: StyleType;
+    /**
+     * Optional CSS classes for the entire dropdown component.
+     */
+    className?: string;
```

Comment:
question: is there a particular reason why `className` needs to be included? We tend to use `style` in our WB components for that purpose, but I'm wondering if we are missing something. Same question applies to `SearchField`.

Reply:
Ah, you're right. I thought it was being used for something else, but it's just styles.